NoteFind.xcodeproj/project.pbxproj

```diff
@@ -67,6 +72,14 @@
 		EE285DC228761B5C00BC7FF0 /* SearchViewController.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = SearchViewController.m; sourceTree = "<group>"; };
 		EE285DC428761B7000BC7FF0 /* FeedViewController.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = FeedViewController.h; sourceTree = "<group>"; };
 		EE285DC528761B7000BC7FF0 /* FeedViewController.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = FeedViewController.m; sourceTree = "<group>"; };
+<<<<<<< Updated upstream
```

Comment:
You'll need to resolve this git merge conflict

here's a resource on the topic: https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts